drivers/metadata/mysql/metadata.go

```diff
@@ -17,6 +17,7 @@ var (
 		infos.WithSequences(false),
 		infos.WithCheckConstraints(false),
 		infos.WithCustomClauses(map[infos.ClauseName]string{
+			infos.ColumnsDataType:                 "column_type",
```

Comment:
I wonder how this worked with `data_type`, or maybe it didn't?

Reply:
It did work with `data_type`, but in mysql there is an additional `column_type` column that already contains the datatype in the format we are looking for.